app/models/user.rb

```diff
@@ -122,6 +122,9 @@ def moderatable
 
   after_save :nullify_authentication_id, if: :any_sso_authorizations?
 
+  after_destroy :update_account_first_admin, if: :account_first_admin?
+  after_create :promote_to_first_admin, if: :should_become_first_admin?
```

Comment:
We should not do that in the user everytime. On Account creation the first admin is always created. See Signup Manager

Reply:
Ok so I will update this in the `Signup Manager` then 😄